service/features/array-config/invalid_multi_az

```diff
@@ -0,0 +1,21 @@
+[
+   {
+      "endpoint": "http://127.0.0.1",
+      "username": "username",
+      "password": "password",
+      "insecure": true,
+      "isDefault": true,
+      "systemID": "14dbbf5617523654",
+      "zone": {
+         "name": "notExistent",
+         "protectionDomains": [
```

Comment:
Is the secret format supposed to be bad or is the zone info supposed to be bad?
I guess it would work either way, right?

Reply:
The zone info is supposed to be bad because we are testing the CreateVolume call with this specifically. The secret is setup for zoning but we do not find a node that valid key so the creation would fail.

Reply:
Oh okay. So in that case, I think we're missing `labelKey`?

Reply:
Although, I thought you'd fixed that already, but maybe it was a different config

Reply:
No, this is supposed to be an invalid_multi_az config, it's supposed to be missing the labelKey and it will default to "" so a topologyZone is not found.

Only used for unit-testing.

Reply:
I see. thanks for clarifying!